src/sfcc/cartridges/int_b2ccrmsync/cartridge/scripts/b2ccrmsync/hooks/customer.process.js

```diff
@@ -18,7 +18,7 @@ var LOGGER = require('dw/system/Logger').getLogger('int_b2ccrmsync', 'hooks.cust
  * @return {Boolean} If the process has been a success or not
  */
 function customerLoggedIn(profile) {
-    if (!require('../util/helpers').isIntegrationEnabled() || !profile) {
+    if (!require('../../util/helpers').isIntegrationEnabled() || !profile) {
```

Comment:
Do we think that helpers.js won't need any customizations? I think this might also need an absolute path

Reply:
What if we replace this line and all other includes of the `helper.js` file with`*/cartridge/scripts/util/helpers` ?